Formula/tssh.rb

```diff
@@ -0,0 +1,21 @@
+class Tssh < Formula
+  desc "SSH Lightweight management tools"
+  homepage "https://github.com/luanruisong/tssh"
+  url "https://github.com/luanruisong/tssh/archive/refs/tags/2.1.0.tar.gz"
+  sha256 "e1086f9cba293bfce8492aef6742ed62f0ed99eb61bf1fc5829b6ef35f5ff881"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPROXY"] = "https://goproxy.cn"
```

Comment:
Please refrain from setting `GOPROXY`. I understand the benefit of using `GOPROXY` when building locally (especially the need for it in China) but from a package development and distribution perspective, we'd prefer not to use it.